src/components/panel/panel.spec.js

```diff
@@ -1053,6 +1053,125 @@ describe('$mdPanel', function() {
     });
   });
 
+  describe('grouping logic:', function() {
```

Comment:
Ping before I delete my comment: It seems like this is with other describes here as well. This is odd.